client/libs/material/src/lib/sermon-table/sermon-table-datasource.ts

```diff
@@ -0,0 +1,68 @@
+import { DataSource } from '@angular/cdk/collections';
+import { MatPaginator } from '@angular/material/paginator';
+import { MatSort } from '@angular/material/sort';
+import { map } from 'rxjs/operators';
+import { merge, Observable, of } from 'rxjs';
+import { Media, Sermon, Speaker } from '@sb/core-data';
+
+type TableTypes = Media[] | Sermon[] | Speaker[] | any;
```

Comment:
We need to include `MediaType` as well.
```suggestion
type TableTypes = Media[] | MediaType[] | Sermon[] | Speaker[] | any;
```